Changed opset.py

SHKIM-Neo opened the pull request:
- Added Identity
> **"test_identity_sequence_cpu"** test is failed.
> This test case was not passed in backend unit tests of ONNX-TF and ONNX-Runtime too. (Refer to details of "ONNX Backend Scoreboard")
> http://onnx.ai/backend-scoreboard/onnx-tf_details_stable.html
> http://onnx.ai/backend-scoreboard/onnxruntime_details_stable.html

- Removed duplicate key "Celu" in opset, argcout, attrset

SHKIM-Neo committed:
Changed opset.py

- Added Identity
> **"test_identity_sequence_cpu"** test is failed.
> This test case was not passed in backend unit tests of ONNX-TF and ONNX-Runtime too. (Refer to details of "ONNX Backend Scoreboard")
> http://onnx.ai/backend-scoreboard/

- Removed duplicate key "Celu" in opset, argcout, attrset

semihlab commented:
I think test case failure is caused by ONNX's bug. Can you put the identity_sequence test to the 'test case prevention list', please?

SHKIM-Neo committed:
Added test to exclude

- The test name 'test_identity_sequence_cpu' was added to test case prevention list.

SHKIM-Neo committed:
Removed PyCharm files
